package.sh

```diff
@@ -27,18 +24,4 @@ docker run --rm \
   -w /cyberark \
   -v $(pwd):/cyberark \
   packager \
-  /bin/bash -c """
-  # Create pkg folders that are the repository of the final artifacts
-  mkdir -p /pkg/bin
-  mkdir -p /pkg/buildpack
-  # Copy the final artifacts to pkg
-  cp manifest.yml CHANGELOG.md CONTRIBUTING.md LICENSE NOTICES.txt README.md VERSION /pkg
-  cp bin/supply bin/compile /pkg/bin
-  cp -R ./buildpack/* /pkg/buildpack
-  # Run buildpack-packager in /pkg
-  pushd /pkg
-    buildpack-packager build -any-stack
-  popd
-  # Move any created zip files from /pkg to the working directory
-  mv /pkg/*.zip .
```

Comment:
was there a benefit to having only a subset of the files in `pkg/`? did it exclude the test files from being packaged in the final artifact in a way they aren't now, or does the packager manifest take care of that?

Reply:
The manifest takes care of that, so only files listed in `manifest.yml` are included. Before, anything listed under `buildpack` would have that same directory structure in the final `.zip`. This would break the offline buildpack, since `bin` would be located under `buildpack`. We essentially duplicated the manifest logic as a workaround, but is now unnecessary.